### DISCUSSION TITLE
Node crashes locally on the second graphql request after updating prisma 4.1.0 -> 4.10.1

### DISCUSSION THREAD

**Discussion by ArtsiomChyrkun** [prisma/prisma#17905] [Q&A]
Hello. I ran into a problem after updating prisma. It doesn't look like prisma related, but updating prisma is only thing I did, so here we are.

I start backend locally like this `dotenv -e .env.dev cross-env NODE_OPTIONS=--max_old_space_size=8192 serverless offline start`. It starts normally, but on the second request process just ends with those errors:
For node v14.21.2
![image](https://user-images.githubusercontent.com/87058485/218418718-d8d46d06-9bde-41b2-bd02-2946d3fd8fd3.png)
For node v17.9.1
![image](https://user-images.githubusercontent.com/87058485/218418757-d303ddf4-e31b-45fe-9a12-340aad5440ef.png)
Looks like something heap related, but I am no sure.
webpack.config.js:
```
const path = require('path');
const slsw = require('serverless-webpack');
const CopyWebpackPlugin = require('copy-webpack-plugin');

const isLocal = slsw.lib.webpack.isLocal;
const entries = {};

Object.keys(slsw.lib.entries).forEach(
    key => (entries[key] = ['./source-map-install.js', slsw.lib.entries[key]])
);

module.exports = {
    mode: isLocal ? 'development' : 'production',
    entry: entries,
    devtool: 'source-map',
    resolve: {
        extensions: ['.mjs','.js', '.jsx', '.json', '.ts', '.tsx'],
        alias: {
            'hiredis': path.join(__dirname, 'aliases/hiredis.js'),
            'superagent-proxy': false,
        }
    },
    plugins: [
        new CopyWebpackPlugin({
            patterns: [
                { from: 'prisma/schema.prisma', to: 'ApolloServer' },
                { from: isLocal ? 'node_modules/.prisma/client/query_engine-windows.dll.node'
                        : 'node_modules/.prisma/client/libquery_engine-rhel-openssl-1.0.x.so.node',
                    to: 'ApolloServer'
                },
            ]}),
    ],
    output: {
        libraryTarget: 'commonjs',
        path: path.join(__dirname, '.webpack'),
        filename: '[name].js',
    },
    target: 'node',
    module: {
        rules: [
            // all files with a `.ts` or `.tsx` extension will be handled by `ts-loader`
            { test: /\.tsx?$/, loader: 'ts-loader' },
            {
                type: 'javascript/auto',
                test: /\.mjs$/,
                use: []
            }
        ],
    },
    optimization: {
        nodeEnv: false,
    }
};
```
I also tried running just `dotenv -e .env.dev serverless offline start`
If you need any additional info, let me know. Thanks in adavance.

**Comment by nurul3101** [comment]
Hello @ArtsiomChyrkun 👋 
Can you check if the error goes away if you downgrade to Prisma 4.1.0?

**Reply by ArtsiomChyrkun** [reply]
Yes, I am at 4.1.0 right now and it works.

**Reply by nurul3101** [reply]
Can you try gradually upgrading and see if the app breaks on a specific version?

**Reply by ArtsiomChyrkun** [reply]
Sure, I can't do it right now, but I'll try every version and post the results here tomorrow.

**Reply by ArtsiomChyrkun** [reply]
Hello @nurul3101 
Sorry for the late response.
I tried step by step upgrading. Used Node version is `v17.9.1`.
I was installing dependencies by running those commands:
```
npm i prisma@version -D
npm i @prisma/client@version
```
And I was testing it by running those commands:
```
npx prisma generate
dotenv -e .env.dev cross-env NODE_OPTIONS=--max_old_space_size=8192 serverless offline start
```
Then I was executing different requests using Apollo studio.
The results are:

- From 4.2.1 to 4.8.1 everything works
- Starting from 4.9.0 it crashes

![image](https://user-images.githubusercontent.com/87058485/218958773-0cd52686-b219-4467-aedc-77ef47598ea1.png)

Also, I am not sure if it can be related to prisma or it only happens in Apollo studio, but sometimes I can get node warnings like this:
![image](https://user-images.githubusercontent.com/87058485/218960202-f82788fc-8e5e-4c5d-99b1-aa9a0db4c54c.png)
![image](https://user-images.githubusercontent.com/87058485/218960217-b4985ee2-ba88-46f3-a74b-af6fc5a5669f.png)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
